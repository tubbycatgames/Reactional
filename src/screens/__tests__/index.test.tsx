import React, { FunctionComponent } from 'react'
import renderer from 'react-test-renderer'
import { View } from 'react-native'

import { withSchemedNav, StackWrapper } from '..'

describe('withSchemedNav', () => {
  it('should wrap a nav stack with scheming params', () => {
    const testView: StackWrapper = ({ screenOptions }) => {
      expect(screenOptions).toMatchSnapshot()
      return <View />
    }
    const SchemedNav = withSchemedNav(testView)
    const render = renderer.create(<SchemedNav />)
    expect(render.toJSON()).toMatchSnapshot()
  })
})
