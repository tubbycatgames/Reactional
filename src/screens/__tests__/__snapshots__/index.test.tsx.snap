// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`withSchemedNav should wrap a nav stack with scheming params 1`] = `
Object {
  "headerStyle": Object {
    "backgroundColor": "#00CFFF",
  },
  "headerTintColor": "#000",
  "headerTitleStyle": Object {
    "fontWeight": "bold",
  },
}
`;

exports[`withSchemedNav should wrap a nav stack with scheming params 2`] = `
<RNCAppearanceProvider
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View />
</RNCAppearanceProvider>
`;
